samcli/lib/telemetry/event.py

```diff
@@ -54,6 +54,9 @@ def __init__(self, event_name: str, event_value: str):
     def __eq__(self, other):
         return self.event_name == other.event_name and self.event_value == other.event_value
 
+    def __repr__(self):
```

Comment:
nit: `__repr__` method usually would provide some developer info, like here return "EventType(event_name, event_value)" maybe better. Not sure how we use this method in our cli.

Reply:
I mostly implemented this to make debugging a little easier, since otherwise it prints something along the lines of `<Event object at 0x12345678>`. Labelling it more clearly would likely give further clarification on the object itself; I'll implement it in a following commit

Reply:
That's a good call out.
Another note - when we embed the events into the metric, we will need to convert the `Event` instances into a json-compatible format

Reply:
Implementing a method to quickly convert the class to something JSON-compatible shouldn't be too tricky. I can implement that as well to append to this PR